angular/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:6.10
+
+MAINTAINER datapunt.ois@amsterdam.nl
+
+EXPOSE 80
+
+
+RUN apt-get update \
+ && apt-get install -y git nginx build-essential openjdk-7-jre \
+ && apt-get clean \
+ && mkdir /app
+
+ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
+
+WORKDIR /app
+COPY *.json /app/
+
+RUN rm -rf node_modules\
+ && yarn install
+
+COPY . /app/
```

Comment:
OK, ik heb hier niet genoeg kennis van om nu makkelijk aan te passen. Dit is voor een vervolg, als ook de acc server wordt ingericht.